### PR TITLE
Update to Manifest 3

### LINKED ISSUE #6: Suddenly not working
The extension icon is greyed out and flashes blue temporarily when clicked on.  I also don't see the the usual option to add to a specific calendar when I right click on an ICS file.  Been using this extension for months so this is new.  Please help.

### PR DESCRIPTION
**Pull request by mbtools**
Thanks for making this nice extension open-source! Let's get this upgrade done 😃 

- Same as #10 but kept the `onbeforeunload` event (needs some testing)
- Minor changes to test cases
- Updated author field
- Added `cache: true` to rollup config since it's [required by rollup-zip](https://github.com/mentaljam/rollup-plugin-zip/issues/9)

Closes #10, #11, #12
Probably closes #6, #7, #9 too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
 
-# Install in Chrome
+## Install in Chrome
+
 You can install the extension directly from the [Chrome Web Store](https://chrome.google.com/webstore/detail/ics-to-gcal/ljobcbehhifehkmamikmchekbbljopao). If you want to make changes to the extension and test it locally, you have to:
 
-1. Run `yarn install` to install dependencies ([ical.js](https://github.com/mozilla-comm/ical.js/) and [chrome-promise](https://github.com/tfoxy/chrome-promise)).
-2. In Chrome, under `More tools -> Extensions`, mark the `Developer mode` checkbox and use `Load unpacked extension...`.
+1. Run `yarn install` to install dependencies ([ical.js](https://github.com/mozilla-comm/ical.js/) and [pure-uuid](https://github.com/rse/pure-uuid)).
+2. Run `yarn run build` to resolve development dependencies (due to changes in Chrome Manifest 3, libraries using CommonJS were the easiest way to restore functionality).
```
Comment (fmeum) [inline README.md:21]: Is the part in brackets more of a comment for me than for the reader? It seems a bit out of place.

**Review by fmeum — COMMENTED** [review]
Thanks for picking this up!

I haven't written any JS lately, but I'll relearn publishing and get out a new version after some testing.

File: README.md (hunk 1 above)
Comment (mbtools) [reply README.md]: That was from the previous PR. I removed it.

**Comment by mbtools** [comment]
AI (cursor) macht's moeglich ;-)

**Pull request merged**